app/modules/kit-library/assets/js/pages/cloud/connect-screen.js

```diff
@@ -33,6 +33,10 @@ export default function ConnectScreen( {
 				elementorCommon.config.library_connect.current_access_tier = data.access_tier;
 				elementorCommon.config.library_connect.plan_type = data.plan_type;
 
+				if ( data.tracking_opted_in && elementorCommon.config.editor_events ) {
```

Comment:
Maybe set it in a function to increase readability?
E.g
```suggestion
				if ( this.isTrackingActive() ) {
					elementorCommon.config.editor_events.can_send_events = true;
```

I would also set this entire condition in a function, I see you use it more than once, this way we won't need to repeat ourself